### PR TITLE
[tutorials] Simplify TMVA_SOFIE_RDataFrame.py.

### PR DESCRIPTION
**Pull request by Axel-Naumann**
Probably doesn't even work...

### CODE REVIEW

File: tutorials/tmva/TMVA_SOFIE_RDataFrame.py [hunk 1]
```diff
@@ -1,62 +1,51 @@
...
 ROOT.TMVA.PyMethodBase.PyInitialize()
 
 
 # check if the input file exists
 modelFile = "Higgs_trained_model.h5"
-if (ROOT.gSystem.AccessPathName(modelFile)) :
-    ROOT.Info("TMVA_SOFIE_RDataFrame","You need to run TMVA_Higgs_Classification.C to generate the Keras trained model")
-    exit()
-
+if (!exists(modelFile)) :
```
Comment (lmoneta) [inline tutorials/tmva/TMVA_SOFIE_RDataFrame.py:21]: `if not exists(modelFile)`

File: tutorials/tmva/TMVA_SOFIE_RDataFrame.py [hunk 2]
```diff
@@ -1,62 +1,51 @@
...
-
+# run inference over input data
+inputFile = "http://root.cern/files/Higgs_data.root"
 df1 = ROOT.RDataFrame("sig_tree", inputFile)
 h1 = df1.Define("DNN_Value", "sofie_functor(rdfslot_,m_jj, m_jjj, m_lv, m_jlv, m_bb, m_wbb, m_wwbb)").Histo1D(("h_sig", "", 100, 0, 1),"DNN_Value")
 
 df2 = ROOT.RDataFrame("bkg_tree", inputFile)
 h2 = df2.Define("DNN_Value", "sofie_functor(rdfslot_,m_jj, m_jjj, m_lv, m_jlv, m_bb, m_wbb, m_wwbb)").Histo1D(("h_bkg", "", 100, 0, 1),"DNN_Value")
 
+# run over the input data once, combining both RDataFrame graphs.
```
Comment (lmoneta) [inline tutorials/tmva/TMVA_SOFIE_RDataFrame.py:46]: `ROOT.RDF.RunGraphs....`

**Review by lmoneta — APPROVED** [review]
Thank you @Axel-Naumann for the PR. 
It works after fixing some C++->Python  syntax in the  script

File: tutorials/tmva/TMVA_SOFIE_RDataFrame.py (hunk 2 above)
Comment (Axel-Naumann) [reply tutorials/tmva/TMVA_SOFIE_RDataFrame.py:47]: `RunGraphs` takes a `vector`, so `[...]` it is!

**Comment by Axel-Naumann** [comment]
@phsft-bot build just on ROOT-ubuntu22.04/default

**Comment by Axel-Naumann** [comment]
@phsft-bot build just on ROOT-ubuntu2204/default

**Review by lmoneta — APPROVED** [review]
LGTM !
Thank you Axel for the improvement in the tutorial!

**Comment by Axel-Naumann** [comment]
Ubuntu warning seems unrelated.

**Comment by hahnjo** [comment]
I believe these commits should have been squashed, the first change is probably not working without the fixup...

**Comment by Axel-Naumann** [comment]
Totally, but I cannot go back in time: this mistake on mine will now be there forever...

**Pull request merged**